### PR TITLE
safe mode screen.

### PR DESCRIPTION
**Pull request by mikehenrty**

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9e8e6003baeb8cc180f48d2a463508ee3149a944

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=157ae6ca06eabd97df7edd5ec5fbf8171f16b42d

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=60d0ec978e05ea683426196799bb698bdfa952c9

File: apps/system/safe_mode.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!doctype html>
```
Comment (KevinGrandon) [inline apps/system/safe_mode.html]: nit: We use `<!DOCTYPE html>` mostly everywhere in gaia, let's keep it consistent.

File: apps/system/safe_mode.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <meta charset="utf-8">
+
+    <link rel="stylesheet" type="text/css" href="shared/style/confirm.css">
+    <link rel="stylesheet" type="text/css" href="shared/style/buttons.css">
+    <link rel="stylesheet" type="text/css" href="style/safe_mode.css">
+
+    <link rel="localization" href="locales/safe_mode.{locale}.properties">
+
+    <script src="shared/js/l10n.js"></script>
+    <script src="js/safe_mode.js"></script>
```
Comment (KevinGrandon) [inline apps/system/safe_mode.html]: nit: maybe add a `defer` statement here?

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=99c010ec0ffae8181c8142e7a0af64f4200e3142

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=767b812a8a4c0b7ce81ad398f83bb1f2c5b64b6b

**Pull request merged**